src/pages/en/index.astro

```diff
@@ -0,0 +1,175 @@
+---
+import config_json from "../../review_templates/template_en.json";
+import PageLayout from "../../layouts/PageLayout.astro";
+---
+
+<PageLayout title="Steam Review Template">
+  <header class="bg-slate-700 text-white py-2 flex flex-col items-center">
+    <h1>{"---{ Review Template }---"}</h1>
+    <a href="/steam-review-template/es/" class="mt-0 text-white">Spanish site</a>
+    <p class="mt-0">☑ Awesome</p>
+  </header>
+
+  <div class="max-w-screen-sm mx-auto px-4">
+    <p>
+      This little tool simplifies the creation of copy-pasta Steam reviews (if
+      you are on this website, this <em>is</em> what you are looking for). I
+      find them moderately funny, so I made this website to avoid copy-pasting
+      the copy pasta so much.&nbsp;😆
+    </p>
+    <p>
+      The code is open source &rarr; <a
+        href="https://github.com/VojtaStruhar/steam-review-template"
+      >
+        check it out on Github</a
+      >!
+    </p>
+    <p>
+      The result generates with a line at the bottom that gives me credit.
+      Leaving it there is optional, but highly appreciated! Thanks!
+    </p>
+    {
+      config_json.categories.map((category) => (
+        <form class="flex flex-col gap-2 ">
+          <fieldset class="rounded-lg my-4 shadow-md border bg-sky-100">
+            <legend class="font-mono font-bold text-xl bg-emerald-300 px-4 rounded-lg shadow">
+              {category.title}
+            </legend>
+            {category.options.map((item) => (
+              <>
+                {category.type == "radio" ? (
+                  <input
+                    type="radio"
+                    id={item}
+                    name={category.title}
+                    value={item}
+                  />
+                ) : (
+                  <input
+                    type="checkbox"
+                    id={item}
+                    name={category.title}
+                    value={item}
+                  />
+                )}
+                <label class="mx-2" for={item}>
+                  {item}
+                </label>
+                <br />
+              </>
+            ))}
+          </fieldset>
+        </form>
+      ))
+    }
```

Comment:
This logic doesn't care about what language it is. We should refactor it into a separate file, so that adding more languages is easier!